difflint/lint_python.py

```diff
@@ -0,0 +1,55 @@
+import pep8
+import pyflakes.api
+
+from .lint_output import LintOutput
+
+
+# We could do this with pep8.StandardReport directly, and its format parameter,
+# but it prints to stdout by default. We don't want that.
+class _PEP8TerseReporter(pep8.BaseReport):
+    """Prints the PEP8 linter results in the expected "terse" format."""
+
+    def __init__(self):
+        super(_PEP8TerseReporter, self).__init__(pep8.StyleGuide().options)
+        self.output = ''
+
+    def error(self, line_number, offset, text, check):
+        code = super(_PEP8TerseReporter, self).error(line_number, offset, text,
+                                                     check)
+        self.output += '{}|{}|{}\n'.format(self.filename, code, text)
+        return code
+
+
+class _PyFlakesTerseReporter:
+    """https://github.com/pyflakes/pyflakes/blob/master/pyflakes/reporter.py"""
```

Comment:
It's not immediately clear to me what the relationship between this file and the code below is.